### DISCUSSION TITLE
Google Calendar not an option

### DISCUSSION THREAD

**Discussion by dmtparker** [kewisch/gdata-provider#446] [Support] [answered]
I am new to this add-on and am probably doing something stupid, but after enabling the add-on and trying to create a new calendar, I only see "On this computer" or "On network" No option to "Google calendar" 
Running Mx Linux, TB version 78.13.0

**Comment by dmtparker** [comment, marked as answer]
OK. I answered my own question, but I'll post it for the next newbie. The wiki says the new calendar dialog should offer "on this computer" "on network" or "Google". It doesn't, just the first two. That was where I was stalled. BUT if you choose "on network" you then on the next screen get to choose "Google" and everything works just fine. Most people would have figured it out before me, but changing the wiki would help folks like me who are too literal.

**Reply by kewisch** [reply]
Thanks for following up! I see you are on an older Thunderbird version, it looks a bit different on Thunderbird 91. I'd strongly recommend upgrading, we also have Thunderbird 102 coming out soon.
